src/EditorFeatures/Core/Shared/Utilities/ClassificationTypeMap.cs

```diff
@@ -23,15 +27,22 @@ internal class ClassificationTypeMap
             // Prepopulate the identity map with the constant string values from ClassificationTypeNames
             var fields = typeof(ClassificationTypeNames).GetFields();
             _identityMap = new Dictionary<string, IClassificationType>(fields.Length, ReferenceEqualityComparer.Instance);
+
             foreach (var field in fields)
             {
-                var value = (string)field.GetValue(null);
-
                 // The strings returned from reflection do not have reference-identity
                 // with the string constants used by the compiler. Fortunately, a call
                 // to string.Intern fixes them.
-                value = string.Intern(value);
-                _identityMap.Add(value, registryService.GetClassificationType(value));
+                var value = string.Intern((string)field.GetValue(null));
+                var classificationType = registryService.GetClassificationType(value);
+                if (classificationType != null)
+                {
+                    _identityMap.Add(value, classificationType);
+                }
+                else
+                {
+                    FatalError.ReportWithoutCrash(new System.Exception($"classification type doesn't exist for {value}"));
```

Comment:
:bulb: Unnecessary `System.` qualification

:bulb: Can we add a `Debug.Assert` here so we get an exception in debug builds?

Reply:
sure